Dockerfile

```
@@ -9,6 +10,7 @@ ENV GOPATH=/go
 COPY --from=golang /go/ /go/
 COPY --from=golang /usr/local/go/ /usr/local/go/
 COPY --from=tfdocs /usr/local/bin/terraform-docs /usr/local/bin/
+COPY --from=terraform /usr/local/bin /usr/local/bin
```

we don't want _all_ of /usr/local/bin, we only want the terraform binary....

```suggestion
COPY --from=terraform /usr/local/bin/terraform /usr/local/bin
```

---

So I actually have a question, I noticed we did that too with terraform-docs..
How do we know where the default installation destination is? I tried to make sense of the process in the Makefile but I don't think I got it...